landing-page/static/css/landing-page.css

```diff
@@ -225,3 +227,43 @@ a.page-scroll {
 .termynal-container {
     padding-top: 3rem;
 }
+
+/* Anchor tags for headers */
+.anchortag { font-size: 80%; visibility: hidden;}
+
+h1:hover a { visibility: visible}
+h2:hover a { visibility: visible}
+h3:hover a { visibility: visible}
+h4:hover a { visibility: visible}
+
+/* Fixed table of contents */
+.grid-container {
+    display: grid;
+    grid-template-columns: 5fr 1fr;
+    grid-gap: 10px;
+}
+
+
+#toc {
+    position: fixed;
+    right: 0;
+    top: 0;
+    background-color:#FFF;
+    display: block;
+    flex: 0 0 20rem;
+    font-size: 0.6rem;
+    width: 20rem;
+    min-width: 0;
+    padding: 30px;
+    padding-right: 50px;

```

Comment:
Why use px on padding?

Reply:
Ah right, that should be `rem`. Updated!